src/core/pretty.ml

```diff
@@ -12,10 +12,60 @@ open Console
 open Pos
 open Syntax
 
+(** [is_keyword s] returns [true] if [s] is a keyword (defined by the lexer)
+    in {!module:Lp_lexer}. *)
+let is_keyword : string -> bool = fun s ->
```

Comment:
Should be moved to lp_lexer.

Reply:
For efficiency, you could use a map or hashtable.

Reply:
I'll see if I can use back the lexer's `match%sedlex` to factorise code.

Reply:
After some thought, I'm not sure which would be the most efficient and the clearer, so I suggest that I leave a note, and we'll optimise in a later pull request.